src/server.ts

```diff
@@ -123,9 +127,10 @@ export async function startServer(options: ServerOptions) {
       // Log server start information
       const enabledToolsCount = availableTools.length;
       const totalToolsCount = TOOLS.length;
-      const logMsg = `Auth0 MCP Server running on stdio with ${enabledToolsCount}/${totalToolsCount} tools available`;
+      const logMsg = `Auth0 MCP Server version ${packageVersion} running on stdio with ${enabledToolsCount}/${totalToolsCount} tools available`;
       logInfo(logMsg);
       log(logMsg);
+      server.sendLoggingMessage({ level: 'info', data: logMsg });
```

Comment:
Now sending this back to client. Very useful for audibility. This is the start of other things to come :)